src/main/resources/templates/db_scripts/migration_scripts/add_job_profile_name.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE ${myuniversity}_${mymodule}.job_executions
```

Comment:
you don't need this anymore right?

Reply:
but you need the job profile Id to be populated ,

Reply:
updated